src/w2utils.js

```diff
@@ -102,6 +102,7 @@ var w2utils = (function ($) {
         escapeId        : escapeId,
         base64encode    : base64encode,
         base64decode    : base64decode,
+        md5encode       : md5encode,
```

Comment:
Since md5 is a one way function, I think it should be called md5(), not md5encode(). I will rename